scripts/game_structure/extended_strenum.py

```diff
@@ -0,0 +1,11 @@
+from strenum import StrEnum
+
+
+class ExtendedStrEnum(StrEnum):
+    @classmethod
+    def list(cls):
+        return list(map(lambda c: c.value, cls))
+
+    @classmethod
+    def index(cls, val: str, **kwargs):
+        return cls.list().index(val)
```

Comment:
Documentation comments, please!

Reply:
Whoops! I'm usually the one asking people to do these, must have slipped through the cracks. Will do that now.